src/FacadeAccountCreation/FacadeAccountCreation.Core/Services/Organisation/OrganisationService.cs

```diff
@@ -279,4 +279,31 @@ public async Task UpdateOrganisationDetails(
 
         response.EnsureSuccessStatusCode();
     }
+
+    public async Task<List<OrganisationNationModel>> GetOrganisationNationByExternalIdAsync(Guid organisationExternalId)
+    {
+        var url = $"{OrganisationNationUrl}?organisationId={organisationExternalId}";
+
+        try
+        {
+            _logger.LogInformation("Attempting to fetch the nation for an organisation id {organisationExternalId} from the backend", organisationExternalId);
+
+            var response = await _httpClient.GetAsync(url);
```

Comment:
as this http call could result in specific status code, consider handing each status code independently 
200, 204, 404, 400, 500, for example 204 is valid non exception response but it wont parse to a list and it .net nullable system might kick up a fuss in future